src/Backend.js

```diff
@@ -16,7 +17,25 @@ export const updateProfile = (username, img, bio) => {
     });
 };
 
-export const getReview = () => {};
+export const getPokemonReviews = (id) =>
+  fetch('/getPokemonReviews', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ pokemonid: id }),
+  })
+    .then((response) => response.json())
+
+export const getUserReview = (username, id) =>
+  fetch('/getUserReview', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ username: username, pokemonid: id }),
+  })
+    .then((response) => response.json())
```

Comment:
Nice, instead of returning the fetch you call fetch. I will modify Profile.js later with this.